bin/index.js

```diff
@@ -8,6 +8,11 @@ const {
   generateSchemaFile
 } = require('../lib/generators');
 
+// Load regular config vars
+require('dotenv').config({
+  path: '.env'
```

Comment:
Q: if developers already have various `.env` files (we have `.env.development` and `.env.production` files), they just need to create a new one, right? I wonder if this `path` can be an optional variable in the `settings` object from `package.json` 😓 sorry for nitpicking lol

Reply:
ohh good idea! can do